store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -8189,6 +8225,55 @@ public void modifyContact(OperationContext octxt, int contactId, ParsedContact p
         }
         return newAddrs;
     }
+    
+    /**
+     * Returns a list of contacts exist in emailed contacts folders.
+     * @param octxt OperationContext object
+     * @param addrs Recipient Email address list
+     * @return listOfContactsToBeModified This list consist list of Map, InternetAddress as key and Pair of ContactId and 
+     * DisplayName as Value
+     * @throws ServiceException 
+     */
+    public List<Map<InternetAddress, Pair<Integer, String>>> findContactsInEmailedContacts
+    (OperationContext octxt, Collection<Address> addrs) throws ServiceException {
+        if (addrs.isEmpty()) {
+            return Collections.emptyList();
+        }
+        List<Map<InternetAddress, Pair<Integer, String>>> list = new ArrayList<>();
+        Map<InternetAddress, Pair<Integer, String>> map = new HashMap<>();
+
+        for (Address addr : addrs) {
+            if (addr instanceof javax.mail.internet.InternetAddress) {
```

Comment:
why are we using complete package `javax.mail.internet.InternetAddress` instead of just `InternetAddress`

Reply:
Done.

Reply:
`javax.mail.internet.InternetAddress` is still being used here

Reply:
One InternetAddress class (from package com.zimbra.common.mime) is already been imported here, so for other same name class I need to type complete package name as prefix to that class in order to use it.

Reply:
ok